packages/apps/transfer/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "./node_modules/@kadena-dev/heft-rig/profiles/default/tsconfig-base.json",
+  "compilerOptions": {
+    "types": ["node"],
+    "target": "ES2020",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "incremental": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "declaration": false, // fixes pnpm issue: https://github.com/microsoft/TypeScript/issues/42873
+    "declarationMap": false,
+    "jsx": "preserve",
+    "paths": {
+      "@/*": ["./src/*"]
+    }
+  },
+  "exclude": ["node_modules"]
+}
```

Comment:
Is this the minimum that we need when based on the `@kadena-dev/heft-rig/profiles/default/tsconfig-base.json`
If it is, and it's required for all next apps that we have, we could make a common `tsconfig-base.json` specifically for `next`. (ie: `@kadena-dev/heft-rig/profiles/nextjs/tsconfig-base.json`

Reply:
new ts config for next is a bit different, I removed duplicates, but the remaining ts config is suitable for new next version